pylint/test/functional/arithmetic_operators.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-docstring,invalid-name,unused-argument
+def sample_function(*args, **kwargs):
```

Comment:
In that case, should we have a separate error message for this which is disabled by default?